### PR TITLE
Add tests for health subsystem configuration

### PR DESCRIPTION
**Pull request by kstekovi**
Add tests for health subsystem configuration

### CODE REVIEW

File: packages/testsuite/cypress/e2e/test-configuration-subsystem-health.cy.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Navigate to Health subsystem", () => {
```
Comment (spriadka) [inline packages/testsuite/cypress/e2e/test-configuration-subsystem-health.cy.ts:19]: Could you please rename it to "toggle security-enabled" as that is the matter of this test?

Comment (kstekovi) [reply packages/testsuite/cypress/e2e/test-configuration-subsystem-health.cy.ts]: sure! renamed to "toggle security-enabled".

**Comment by kstekovi** [comment]
@spriadka GH action also updated. Could you please take a second round or review?

**Comment by spriadka** [comment]
@kstekovi Looks good to me, merging!. Thank you for the contribution 🚀

**Pull request merged**